src/Libraries/RevitNodes/Elements/Parameter.cs

```diff
@@ -40,5 +251,193 @@ public override string ToString()
             }
             return string.Format("{0} : {1}", Name, value);
         }
+
+        #endregion
+
+        #region Shared Parameters
+
+        /// <summary>
+        /// Get shared parameter file path
+        /// </summary>
+        /// <returns></returns>
+        public static string SharedParameterFile()
+        {
+            Document document = Application.Document.Current.InternalDocument;
+            return document.Application.SharedParametersFilename;
+        }
+
+        /// <summary>
+        /// Create Shared Parameter
+        /// </summary>
+        /// <param name="parameterName">Name</param>
+        /// <param name="groupName">Group of the parameter for shared parameters</param>
+        /// <param name="type">Parameter Type</param>
+        /// <param name="group">Parameter Group</param>
+        /// <param name="instance">Is instance parameter, otherwise its a type parameter</param>
+        /// <param name="categoryList">List of categories this parameter applies to</param>
+        public static void CreateSharedParameter(string parameterName, string groupName, string type, string group, bool instance, System.Collections.Generic.List<Category> categoryList = null)
```

Comment:
It will be good to have a node to allow users to choose group name and type. Or whoever use this node will have to figure out this.